pkg/ovsdb/dbserver.go

```diff
@@ -19,14 +19,25 @@ import (
 	"github.com/ibm/ovsdb-etcd/pkg/types/_Server"
 )
 
+type DBServerInterface interface {
+	Lock(ctx context.Context, id string) (bool, error)
+	Unlock(ctx context.Context, id string) error
+	AddSchema(schemaName, schemaFile string) error
```

Comment:
AddSchema and especially LoadServerData are part of the development scaffolding, they will be removed.

Reply:
but this is not related to the focus of this PR, this is future work which should be done but should not be mixed with this effort